css/stylesheet.css

```diff
@@ -0,0 +1,803 @@
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-content: flex-start;
+  align-items: center;
+  margin-left: 30%;
+  margin-right: 30%;
+}
+.port-navbar a {
+  display: inline-block;
+  text-align: center;
+  font-size: 1em;
+  font-family: "Droid sans", sans-serif;
+  text-decoration: none;
+  color:#a3a3a3;
+}
+.port-navbar a:hover {
+  background-color: red;
+}
+.icon-menu {
+    display:inline-block;
+}
+nav ul, nav:active ul {
+  display: none;
+}
+nav li {
+  text-decoration: none;
+  text-align: left;
+  width: 100%;
+  padding: 10px 0;
+}
+nav:hover ul {
+   display: block;
+}
+
+.protfolio_pieces{
+  display:flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: flex-start;
+  margin-bottom: 4em;
+  text-align: center;
+}
+.l-icon-outer {
+  position: relative;
+  margin: auto;
+  border-radius: 50%;
+  height: 75px;
+  width: 75px;
+  background-color: #fc6432;
+}
+.l-droplet-icon {
+  position: absolute;
+  left: 25px;
+  top: 25px;
+  margin: auto;
+  width: 25px;
+  height: 25px;
+  overflow: hidden;
+  border-bottom-right-radius: 50%;
+  border-bottom-left-radius: 50%;
+  border-top-left-radius: 50%;
+  -webkit-transform:rotate(-45deg);
+  z-index: 99;
+  background: white;
+}
+.l-droplet-icon-interior {
+  position: absolute;
+  left: 2.5px;
+  top: 2.5px;
+  margin: auto;
+  width: 20px;
+  height: 20px;
+  overflow: hidden;
+  border-bottom-right-radius: 50%;
+  border-bottom-left-radius: 50%;
+  border-top-left-radius: 50%;
+  z-index: 150;
+  background: #fc6432;
+}
+ul {
+  list-style-type: none;
+  padding-left: 0;
+}
+.l-home-spacer {
+  height: 4em;
+}
+.l-home-text {
+  font-weight: lighter;
+  margin: auto;
+  width: auto;
+  text-align: center;
+  font-family: "Droid sans";
+}
+.l-pricing-tables {
+  margin-bottom: 0;
+}
+h2 {
+  font-size: 1.6em;
+}
+h3 {
+  font-size: 1.0em;
+  font-family: "Droid sans", sans-serif, sans-serif;
+}
+p {
+  font-family: "Droid sans", sans-serif, sans-serif;
+  font-size: 1em;
+}
+footer h1{
+  margin: .9px;
+}
+.p-divet {
+  width: 0;
+  height: 0;
+  border-top: 20px solid #fc6432;
+  border-left: 80px solid transparent;
+  border-right: 80px solid transparent;
+  margin: auto;
+}
+.p-title-line{
+    border-bottom: 1px white solid;
+    width: 15%;
+}
+
+.p-title{
+  text-transform: uppercase;
+  padding-left: 4%;
+  padding-right: 4%;
+}
+.p-title-line_gray{
+  color: black;
+  border-bottom: 1px #bfbfbf solid;
+  width: 15%;
+}
+.p-title-line_little{
+  border-bottom: 1px white solid;
+  width: 10%;
+}
+.p-title-line_little_gray{
+  color:black;
+  border-bottom: 1px gray solid;
+  width: 10%;
+}
+
+.l-title{
+  display:flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+.p-title{
+  text-transform: uppercase;
+  padding-left: 4%;
+  padding-right: 4%;
+}
+
+.l-team {
+  text-align: center;
+}
+.l-team-member {
+  margin-bottom: 10%;
+}
+.p-team h4 {
+color: #fc6432;
+}
+.l-client-link {
```

Comment:
Random selectors showing up in the document. As this document grows, the harder it is to maintain if there is no clear code grouping.